### DISCUSSION TITLE
Letterboxing Is broken after update

### DISCUSSION THREAD

**Discussion by moshier1** [ppy/osu#15476] [Q&A]
Screen scaling no longer has the background on the outer bars, it's now black. In game there are now two progress bars.
![osu_2021-11-04_15-50-52](https://user-images.githubusercontent.com/9275252/140411438-f262d192-c771-4b75-85c3-300dee449375.png)
![osu_2021-11-04_15-53-36](https://user-images.githubusercontent.com/9275252/140411446-fbbf0dce-b191-4caf-969c-020f650096d7.png)
[database.log](https://github.com/ppy/osu/files/7478142/database.log)
[network.log](https://github.com/ppy/osu/files/7478144/network.log)
[performance.log](https://github.com/ppy/osu/files/7478145/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7478146/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7478147/updater.log)

**Comment by bdach** [comment]
background is another case of https://github.com/ppy/osu/issues/15475.

unsure what the skin issue is about and it's a separate issue. please export and upload the skin.

**Reply by moshier1** [reply]
Didn't know that was related to the skin, here's the skin, changed extension to zip to upload. I had the default one on previously, it works after changing it back to default in this version. Thanks.
[osu!lazer (modified) (team osu!).zip](https://github.com/ppy/osu/files/7478228/osu.lazer.modified.team.osu.zip)

**Reply by peppy** [reply]
You can fix the skin by deleting one progress bar via the skin layout editor.
